client.go

```diff
@@ -99,6 +99,7 @@ func newClient(endpoint string, isProtobuf bool, config Config) *Client {
 	if config.Name == "" {
 		config.Name = "go"
 	}
+
```

Comment:
Should we also set `config.Proxy` to `http.ProxyFromEnvironment` if it is `nil`, to maintain backwards compatible behaviour? 🤔

Reply:
Hmm, yes to maintain backwards compatibility. Lemme make this change. This would mean that it would be impossible to disable proxies, but :shrug: that doesn't seem to be a complaint